docs/sdk/client-side/android/index.md

```diff
@@ -116,31 +117,35 @@ The initialize process starts polling the latest variations from Bucketeer in th
 
 If you want to use the feature flag on Splash or Main views, and the user opens your application for the first time, it may not have enough time to fetch the variations from the Bucketeer server.
 
-For this case, we recommend using the callback in the initialize method.
+For this case, we recommend using the `Future<BKTException?>` returned from the initialize method.
 
 <Tabs>
 <TabItem value="kt" label="Kotlin">
 
 ```kotlin showLineNumbers
-val callback = object : BKTClientInterface.FetchEvaluationsCallback {
-  override fun onSuccess() {
-    val showNewFeature = client.booleanVariation("YOUR_FEATURE_FLAG_ID", false)
+// The callback will return without waiting until the fetching variation process finishes
+val timeout = 1000 // Default is 5 seconds
+
+viewLifecycleOwner.lifecycleScope.launch {
+  val future = BKTClient.initialize(this.application, config, user, timeout)
+
+  // Future is blocking, so you need to wait on non-main thread.
+  val error = withContext(Dispatchers.IO) {
+    future.get()
+  }
+  
```

Comment:
Please delete the space at the beginning of this line.

Reply:
updated: [`f979f73` (#6)](https://github.com/bucketeer-io/bucketeer-docs/pull/6/commits/f979f73ee3689e16a8b1e03b8c76de20fed952fb)